src/main/java/ce/ajneb97/managers/EventsManager.java

```diff
@@ -159,118 +163,249 @@ private CheckConditionsResult checkConditions(ConditionEvent conditionEvent, boo
-
-                        conditionLineWithReplacements = conditionLineWithReplacements+"'"+arg1+"'"+textToFind+"'"+arg2+"'";
-                        if(c != orConditions.length-1){
-                            conditionLineWithReplacements = conditionLineWithReplacements+" or ";
+            if (conditionLine.contains(" and ")) {
+                String[] andConditions = conditionLine.split(" and ");
+                for(int c=0;c<andConditions.length;c++){
+                    boolean miniConditionApproved = false;
+                    //If a miniCondition is NOT accomplished, it will check the next.
+                    //If a miniCondition IS accomplished, it will finish this cycle.
+                    //If this cycle is finished with NO accomplished miniConditions, the
+                    // whole condition was not satisfied so the checkConditions method returns FALSE.
+                    String miniCondition = andConditions[c];
+
+                    for(ConditionalType conditionalType : ConditionalType.values()){
+                        String textToFind = " "+conditionalType.getText()+" ";
+                        if(miniCondition.contains(textToFind)){
+                            int textToFindIndex = miniCondition.indexOf(textToFind);
+                            String arg1 = miniCondition.substring(0, textToFindIndex);
+                            String arg2 = miniCondition.substring(textToFindIndex+conditionalType.getText().length()+2);
+                            //Replace variables
+                            VariablesProperties variablesProperties = new VariablesProperties(
+                                    storedVariables,player,target,isPlaceholderAPI,event,minecraftEvent
+                            );
+                            arg1 = VariablesUtils.replaceAllVariablesInLine(arg1,variablesProperties,false);
+                            arg2 = VariablesUtils.replaceAllVariablesInLine(arg2,variablesProperties,false);
+
+                            conditionLineWithReplacements = conditionLineWithReplacements+"'"+arg1+"'"+textToFind+"'"+arg2+"'";
+                            if(c != andConditions.length-1){
+                                conditionLineWithReplacements = conditionLineWithReplacements+" and ";
+                            }
+
+                            String firstArg = !mathFormulas ? arg1 : MathUtils.calculate(arg1);
+                            String secondArg = !mathFormulas ? arg2 : MathUtils.calculate(arg2);
+
+                            String firstArgLower = firstArg.toLowerCase();String secondArgLower = secondArg.toLowerCase();
+                            double firstArgNum = 0;
+                            double secondArgNum = 0;
+                            try{
+                                firstArgNum = Double.parseDouble(firstArg);
+                                secondArgNum = Double.parseDouble(secondArg);
+                            }catch(NumberFormatException ignored){}
```

Comment:
One of the libraries Spigot provides in their API (Guava) has [Doubles#tryParse(String)](https://guava.dev/releases/19.0/api/docs/com/google/common/primitives/Doubles.html#tryParse(java.lang.String)). There’s also [NumberUtils#toDouble(String)](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/math/NumberUtils.html#toDouble-java.lang.String-) which is provided by Apache Commons but the library has been removed from the API in newer versions.